from error import setup_NTL_error_callback
setup_NTL_error_callback()
